lms/static/js/instructor_dashboard/membership.js

```diff
@@ -138,13 +143,15 @@ such that the value can be defined later than this assignment (file load order).
         };
 
         AuthListWidget.prototype.reload_list = function() {
-            var authlistwidgetreloadlist = this;
+            var authlistwidgetreloadlist = this,
+                $selectedOption;
             return this.get_member_list(function(error, memberList) {
                 if (error !== null) {
-                    return authlistwidgetreloadlist.show_errors(error);
+                    authlistwidgetreloadlist.show_errors(error);
```

Comment:
Either restore the return, or ensure that nothing nefarious happens by continuing execute.